README.md

```diff
@@ -81,9 +78,9 @@ String fingerprint = fingerprintAndroidAgent.getFingerprint(
 String deviceId = fingerprintAndroidAgent.deviceId();
 
 ```
-#### 2. Advanced
+#### Advanced usage
 
-Agent has configuration class. Use it, when your preferred version of fingerprints or hash type distinct from default.
+Agent has a configuration class. Use it when your preferred version of fingerprints or hash type are distinct from default.
 
 ```kotlin
 data class FingerprintAndroidConfiguration(
```

Comment:
this class name looks redundant to me, is there a concept of namespace? We could simplify like follows (in pseudo code)

```
import Configuration from AndroidFingerprint
// use Configuration here
```

Should probably be `AndroidFingerprint`, not `FingerprintAndroid`. FingerprintAndroid is a human-like robot that takes your fingerprints when you cross a border. `AndroidFingerprint` is a library name for android platform.

Reply:
> Use it when your preferred version of fingerprints or hash type are distinct from default.

Unclear what that means

Reply:
and we probably don't need to have the word "android" at all here, it's obvious that it's android.